Week-4/Javascript_basics/Closures/script.js

```diff
@@ -0,0 +1,19 @@
+const Counter = () => {
+  let Count = 1;
```

Comment:
```suggestion
  let count = 1;
```
only classNames should be capitalized in JS

Reply:
Hmm fine